docs_theme/main.html

```diff
@@ -0,0 +1,57 @@
+{% extends "base.html" %}
+
+{% block scripts %}
+{{ super() }}
+<script>
+  fetch('{{ 'versions.txt'|url }}')
+    .then(function(res) {if (res.ok) return res.text();})
```

Comment:
Funny to see you using the function syntax, most JS I see uses anonymous functions for their callbacks.

I think you need to also supply the onRejected argument for these promises incase there is a network error.

Reply:
This is an anonymous function. It does not have a name.
What are we supposed to do in the rejected code path? I think, we just don't add the widget to the dom tree.

Reply:
Woops, meant the arrow syntax is what I always see. 

Yeah probably do nothing on error. I guess we probably don't have to add a handler.